admin-dev/themes/new-theme/js/pages/order/view.js

```diff
@@ -160,4 +160,9 @@ $(() => {
       $modal.find(OrderViewPageMap.updateOrderAddressTypeInput).val($btn.data('address-type'));
     });
   }
+
+  $(`${OrderViewPageMap.displayPartialRefundBtn}, ${OrderViewPageMap.cancelPartialRefundBtn}`).on('click', (event) => {
+    event.preventDefault();
```

Comment:
Since it's now a button no need for `event.preventDefault();` any more

Reply:
done